torch/autograd/profiler_util.py

```diff
@@ -485,6 +486,9 @@ def __init__(
         self.sequence_nr: int = sequence_nr
         self.device_type: DeviceType = device_type
         self.device_index: int = device_index
+        self.device_resource_id: int = (
+thread if device_resource_id is None else device_resource_id
```

Comment:
This need proper indentation, around 12 spaces.
```suggestion
            thread if device_resource_id is None else device_resource_id
```